### PR TITLE
cloudlogoffline: init at 1.1.4

### PR DESCRIPTION
**Pull request by dblsaiko**
## Description of changes

Adds cloudlogoffline, a companion app to [Cloudlog](https://www.magicbug.co.uk/cloudlog/) (pkgs.cloudlog)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/cl/cloudlogoffline/package.nix
Comment (Janik-Haag) [inline pkgs/by-name/cl/cloudlogoffline/package.nix:44]: please add meta.mainProgram

**Review by Janik-Haag — COMMENTED** [review]
Please add a desktop item so stuff like rofi, fuzzel, etc.. pick this up since it's a desktop application.

**Comment by 2xsaiko** [comment]
> Please add a desktop item so stuff like rofi, fuzzel, etc.. pick this up since it's a desktop application.

Turns out I accidentally nuked that when I did the last change from another computer and forgot to pull the branch. I redid it better than before though!

Is there a way to test desktop entries in packages without adding them to systemPackages?

**Comment by Janik-Haag** [comment]
> Is there a way to test desktop entries in packages without adding them to systemPackages?

nope sadly not aware of anything.

This should be good to go once you rebase and drop your maintainer commit since it just landed in master because I merged your other pr.

**Comment by Janik-Haag** [comment]
@ofborg build cloudlogoffline

**Review by Janik-Haag — APPROVED** [review]

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 270713` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cloudlogoffline</li>
  </ul>
</details>

**Pull request merged**
